### DISCUSSION TITLE
9E3D70C2-8DF0-590F-2EB4-2356E78D9C28

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#37] [General]
# NGC 7293 (Helix Nebula)

The Helix Nebula, also known as NGC 7293 or Caldwell 63, is a planetary nebula located about 655 light-years away in the constellation Aquarius. This nebula has sometimes been referred to as the "Eye of God" in pop culture, as well as the "Eye of Sauron"

https://celestia.mobi/addon?item=9E3D70C2-8DF0-590F-2EB4-2356E78D9C28

**Comment by Pascualito1008** [comment]
Not ring nebula

**Reply by DaveBowman2001** [reply]
The Ring Nebula is a different object from Helix. It is labeled as [Messier 57](https://celestia.mobi/resources/item?item=26AE2F6D-406F-ACE1-617F-9C3364263CCC) in the Nebulae add-on category

**Reply by kmjordan122313-lgtm** [reply]
this is one of the best add-ons.

**Comment by dave-kaye** [comment]
Starting 6/25, Celestia 1.7.0 has supported nebula types.

To use this feature, add the following two lines to the first data grouping in the file helix_nebula.dsc

    Type  "Planetary"
    AbsMag  1.10

Data from Wikipedia - Helix Nebula.
